.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -42,37 +40,23 @@ function updateNativeVersions(version) {
     }
 }
 
-const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
 let semanticVersionLevel = core.getInput('SEMVER_LEVEL', {require: true});
-
-if (!semanticVersionLevel || !_.find(versionUpdater.SEMANTIC_VERSION_LEVELS, v => v === semanticVersionLevel)) {
+if (!semanticVersionLevel || !_.contains(versionUpdater.SEMANTIC_VERSION_LEVELS, semanticVersionLevel)) {
     console.log(
         `Invalid input for 'SEMVER_LEVEL': ${semanticVersionLevel}`,
         `Defaulting to: ${versionUpdater.SEMANTIC_VERSION_LEVELS.BUILD}`,
     );
     semanticVersionLevel = versionUpdater.SEMANTIC_VERSION_LEVELS.BUILD;
 }
-console.log('Fetching tags from github...');
-octokit.repos.listTags({
-    owner: github.context.repo.owner,
-    repo: github.context.repo.repo,
-})
-    .catch(githubError => core.setFailed(githubError))
-    .then((githubResponse) => {
-        // Find the highest version git tag
-        const tags = githubResponse.data.map(tag => tag.name);
 
-        // tags come from latest to oldest
-        const highestVersion = tags[0];
+const {version: previousVersion} = JSON.parse(fs.readFileSync('./package.json'));
+const newVersion = versionUpdater.incrementVersion(previousVersion, semanticVersionLevel);
+console.log(`Previous version: ${previousVersion}`, `New version: ${newVersion}`);
 
-        console.log(`Highest version found: ${highestVersion}.`);
+updateNativeVersions(newVersion);
 
-        newVersion = versionUpdater.incrementVersion(highestVersion, semanticVersionLevel);
-
-        updateNativeVersions(newVersion);
-        console.log(`Setting npm version for this PR to ${newVersion}`);
-        return exec(`npm --no-git-tag-version version ${newVersion} -m "Update version to ${newVersion}"`);
-    })
+console.log(`Setting npm version to ${newVersion}`);
+exec(`npm --no-git-tag-version version ${newVersion} -m "Update version to ${newVersion}"`)
```

Comment:
You can just do `"Update version to %s"` here. ([ref](https://docs.npmjs.com/cli/v7/commands/npm-version))